specification/network/resource-manager/Microsoft.Network/2017-06-01/expressRouteConnector.json

```diff
@@ -0,0 +1,372 @@
+					  "$ref": "#/definitions/expressRouteConnector"
+					}
+				  }
+				},
+				"x-ms-long-running-operation": true
+			}
+		},
+	 "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/expressRouteConnectors": {
+		  "get": {
+			"tags": [
+			  "ExpressRouteConnectors"
+			],
+			"operationId": "ExpressRouteCircuitConnectors_ListByResourceGroup",
+			"x-ms-examples": {
+                    "ExpressRouteConnectorListByResourceGroup": { "$ref": "./examples/ExpressRouteConnectorListByResourceGroup.json" }
+            },
+			"description": "Gets all express route Connector resources in a given resource group and subscription.",
+			"parameters": [
+			  {
+				"$ref": "#/parameters/ResourceGroupNameParameter"
+			  },
+			  {
+				"$ref": "#/parameters/ApiVersionParameter"
+			  },
+			  {
+				"$ref": "#/parameters/SubscriptionIdParameter"
+			  }
+			],
+			"responses": {
+			  "200": {
+				"description": "Request successful. The operation returns a list of Express Route Connector resources.",
+				"schema": {
+				  "$ref": "#/definitions/expressRouteConnectorListResult"
+				}
+			  }
+			},
+			"x-ms-pageable": {
+				"nextLinkName": "nextLink"
+			}
+		  }
+		},	
+	"/subscriptions/{subscriptionId}/providers/Microsoft.Network/expressRouteConnectors": {
+		  "get": {
+			"tags": [
+			  "ExpressRouteConnectors"
+			],
+			"operationId": "ExpressRouteCircuitConnectors_ListBySubscription",
+			"x-ms-examples": {
+                    "ExpressRouteConnectorListBySubscription": { "$ref": "./examples/ExpressRouteConnectorListBySubscription.json" }
+            },
+			"description": "Gets all express route Connector resources in a given subscrption.",
+			"parameters": [
+			  {
+				"$ref": "#/parameters/ApiVersionParameter"
+			  },
+			  {
+				"$ref": "#/parameters/SubscriptionIdParameter"
+			  }
+			],
+			"responses": {
+			  "200": {
+				"description": "Request successful. The operation returns a list of Express Route Connector resources.",
+				"schema": {
+				  "$ref": "#/definitions/expressRouteConnectorListResult"
+				}
+			  }
+			},
+			"x-ms-pageable": {
+				"nextLinkName": "nextLink"
+			}
+		  }
+		}	
+	},	 
+	"definitions": {	
+		"expressRouteConnectorPropertiesFormat": {
+			"required": [
+				"circuits",
+				"addressPrefix"
+			],
+			"properties": {
+				"circuits": {
+					"type": "array",
+					"items": {
+						"$ref": "./network.json#/definitions/SubResource",
+						"description": "The reference to the express route circuit resource."
+					},
+					"description": "List of Express Route Circuits."
+				},
+				"authorizations": {
+					"type": "array",
+					"items": {
+						"type": "string",
+						"description": "Authorization Key"
+					},
+					"description": "List of Circuit Authorization Keys."
+				},
+				"addressPrefix": {
+				  "type": "string",
+				  "description": "The reference of the addressPrefix resource."
+				},
+				"connectionStatus": {
```

Comment:
I checked with team internally here and for now we just need provisioningState. When we expand the API to accept list of circuits we would consider exposing connectionStatus for each link between a pair of circuits